### PR TITLE
MLR 503 local SNS and S3 via localstack

### PR DESCRIPTION
**Pull request by cschroed-usgs**
This PR proposes using localstack to provide a local SNS. localstack is likely to be a sustainable solution for local AWS emulation in the long term. It has 27k stars on GitHub. Development will be funded by licensing for premium features, and via individual and corporate donations through opencollective. The free version of localstack provides several AWS services, including SNS and S3. This PR proposes eliminating the mock S3 container in favor of using localstack's S3 implementation. I also considered adopting other unrelated standalone docker images that mock the SNS API. While that works, I thought it would be easier to use a single consolidated solution for mocking AWS services.

### CODE REVIEW

File: docker-reference/configuration/mlr-wsc-file-exporter/config.env [hunk 1]
```diff
@@ -3,8 +3,5 @@ CERT_IMPORT_DIRECTORY=/home/python/certificates
 ssl_cert_path=/home/python/certificates/ssl.crt
 ssl_key_path=/home/python/certificates/ssl.key
 S3_BUCKET=mock-bucket
-AWS_REGION=default-region
 TIERNAME=test
 S3_ENDPOINT_URL=http://localhost:8080
-AWS_ACCESS_KEY_ID=remote-identity
-AWS_SECRET_ACCESS_KEY=remote-credential
```
Comment (cschroed-usgs) [inline docker-reference/configuration/mlr-wsc-file-exporter/config.env:10]: These `AWS` vars were moved to `common/config.env`

**Review by zmoore-usgs — APPROVED** [review]
Very cool!

Only feedback might be to add to the readme that the AWS command-line will be needed on the host machine in order to create the SNS topic and S3 bucket. We're getting to the point where having the AWS CMD installed locally is more of an assumption (like Docker) but not sure we're there quite yet.

**Comment by cschroed-usgs** [comment]
Good point @zmoore-usgs ! I added that to the README.

**Comment by kmschoep-usgs** [comment]
Looks good! It will be an adventure when I run this for the first time, I've not had great luck with saml2aws work very smoothly.

**Comment by cschroed-usgs** [comment]
👍 The struggle is real. I will send you a few links for saml2aws. Fortunately, the aws cli commands in the backing services script are interacting with an unauthenticated local mock of the AWS api, so you should be able to stand it up without resolving saml2aws issues!

**Pull request merged**